src/main/resources/org/jboss/xavier/analytics/rules/workload/inventory/BasicFields.drl

```diff
@@ -9,14 +9,29 @@ auto-focus true
 
 rule "Copy basic fields and agenda controller"
     when
-        foo: VMWorkloadInventoryModel(
-            // add conditions
+        vmWorkloadInventoryModel : VMWorkloadInventoryModel(
+            provider != null ,
+            datacenter != null,
+            cluster != null,
+            vmName != null,
+            diskSpace != null,
+            memory != null,
+            cpuCores != null,
+            guestOSFullName !=null
```

Comment:
Please add `osProductName` as well and copy its value to the `osName` field, updating also the tests